templates/docs/patterns/hero/index.md

```diff
@@ -17,12 +17,105 @@ The hero pattern is composed of the following elements:
 
 | Element              | Description                                                              |
 | -------------------- | ------------------------------------------------------------------------ |
-| Title (**required**) | <code>h1</code> title text                                               |
-| Subtitle             | <code>h2</code> subtitle text                                            |
-| Description          | <code>p</code> description text                                          |
+| Title (**required**) | `h1` title text                                                          |
+| Subtitle             | `h2` subtitle text                                                       |
+| Description          | `p` description text                                                     |
```

Comment:
It is a slot where we expect free HTML right? This suggest (similar to titles above) that we expect a text, that will be put inside a paragraph. So the addition of `p` seems to make it a bit misleading.

Reply:
Hero macro itself does not place any paragraphs there - it just wraps whatever you pass in in a shallow section ([src](https://github.com/canonical/vanilla-framework/blob/68ea7039cf74936fd19ae6d7802c1eb7498f83dd/templates/_macros/hero.jinja#L105-L111))

I think the change you already suggested I make (to change the wording to "Description text (one or more paragraphs") covers this.